Formula/b/blisp.rb

```diff
@@ -0,0 +1,23 @@
+class Blisp < Formula
+  desc "ISP tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs"
+  homepage "https://github.com/pine64/blisp"
+  url "https://github.com/pine64/blisp.git",
+      tag:      "v0.0.4",
+      revision: "f601b6b965a922b4a9f400d1c8fd946b69708728"
```

Comment:
looking at the submodule, we can refactor to have the formula depends on argtable3 and libserialport